Model/CompanyRepository.php

```diff
@@ -0,0 +1,179 @@
+<?php
```

Comment:
Good practice to include declare(strict_types=1); at the top of php files. Check code for some examples